### ISSUE TITLE
Giving ooc to the community

### ISSUE THREAD

**Issue by fasterthanlime** [ooc-lang/rock#972]
Here's the short of it:

  * I became too busy to take care of ooc
  * I tried to keep maintaining rock & other tools, and tried to keep the original philosophy/spirit
  * That kinda failed
  * Cogneco/imint folks reached out, asked me how I felt about paid bounties to work on their rock fork
  * My pov is that I see no reason to maintain two forks, and if they're the primary users they should have enough latency to push it forward even when I'm afk

So, I've started reorganizing the project so that it can be developed, documented, driven and released by people other than me.

The first step was transferring repos to a Github organization (`ooc-lang`), more steps will follow. In particular:

  * Releases could be hosted on Github releases (a good setup would be Travis-CI+dpl)
  * The website could be hosted on Github pages (instead of my own Amazon S3 bucket)

I'm aware of the risks of having a company control the fate of an open source project and that's why I want to make sure there are safeguards, but I also feel like I've been hindering the development of rock and I'd like to stop doing that.

As far as I can tell, ooc is a core part of cogneco/imint's business, and I can't think of a better motivator for them to keep it solid & useful, even if it does drift away from my initial ideals a little.

**Comment by vendethiel** [comment]
:+1:

**Comment by kirbyfan64** [comment]
:(

(cues [Taps](https://en.m.wikipedia.org/wiki/Taps))

**Comment by fasterthanlime** [comment]
Note that this isn't a sad thing! Instead of having ooc slowly die from my lack of involvement, it can thrive under someone else's direction. I think it's pretty good news, I just failed to frame it in such a light :)

**Comment by zhaihj** [comment]
:+1:

**Comment by davidhesselbom** [comment]
Actually, I no longer work for Imint, and while I really meant it when I said I like ooc, I don't have any time to spend on it now that I'm no longer paid to do so. :-/ I also think a few things are missing from the language, things that I suspect are intentionally left out and that I suspect the people at Imint will want to add, and that they didn't think you would ever approve of (hence the fork).

If you want the Imint people in on this, I think you may want to add https://github.com/fredrikbryntesson and/or https://github.com/thomasfanell to this group. They were the ones mostly involved with modifying rock - all I did was report bugs and request features :)

If there's one thing I would like to contribute with, if I can it's setting up AppVeyor to build Windows releases. Making rock work in Windows was way harder than it should be, at least the last time I tried.

**Comment by shamanas** [comment]
I agree completely with Amos here.

**Comment by fasterthanlime** [comment]
So, I've had news from Fredrik, he wasn't expecting my offer to have 'their' fork become the main one, and he says their fork is not ready (some of the changes aren't "clean enough" to be merged back in the main tree, etc.) so it looks like, for now, they'll continue as two separate projects.

**Comment by fredrikbryntesson** [comment]
Yes as Amos said we should keep the forks separate. But I hope and guess that almost everything we will do in rock will be easy to merge into upstream. I hope that our bounty program will contribute to both our fork and ooc-lang :)
